packages/db/src/translate/granules.ts

```diff
@@ -179,6 +190,10 @@ export const translateApiGranuleToPostgresGranule = async (
     updated_at: new Date(dynamoRecord.updatedAt),
   };
 
+  if (preserveNilProperties) {
```

Comment:
I'm wondering if instead of this pattern we should have a differently-named wrapper that does this after calling `translateApiGranuleToPostgresGranule`.  

See comments above, let me know what you think.

Reply:
Yup, I like it. https://github.com/nasa/cumulus/pull/3082/commits/9c8a70e1e4bedc19e7da5dd2467c079030590630